src/pages/Homepage.js

```diff
@@ -0,0 +1,17 @@
+import '../css/home-page.css';
+import { HomePageCategories, HomePageCards, HomePageHero } from '../components/Homepage';
+import { Navigation } from '../components/Navigation';
+
+const HomePage = () => {
+
+    return <div className='home-page-body bg-clr'>
+        <Navigation />
+        <HomePageCategories />
+        <HomePageHero />
+        <HomePageCards />
+        </div>
+    
+    
```

Comment:
Just minor suggestion: you can format document later on to make it cleaner

Reply:
I tried using Beautify and Prettier, it messes with code.. will look into other options. Feel free to suggest if you have any.